src/bridge.js

```diff
@@ -14,19 +14,25 @@ type Message = {|
 export default class Bridge extends EventEmitter {
   _messageQueue: Array<any> = [];
   _timeoutID: TimeoutID | null = null;
+  _destroyed: boolean = false;
```

Comment:
subjective nit: Not a fan of the name `_destroyed`. I think `isShutdown` would be a bit more consistent with other similar var names within the repo.

Reply:
Yes, leftover from when the shutdown function was called destroy. It's become a muscle memory for me to write these destroy/_destroyed patterns.